package.json

```diff
@@ -1,10 +1,11 @@
 {
   "dependencies": {
-    "@ckeditor/ckeditor5-build-classic": "^11.0.1",
-    "@rails/webpacker": "3.5",
+    "@ckeditor/ckeditor5-build-classic": "12.3.1",
+    "@rails/webpacker": "4.0.7",
+    "core-js": "3",
     "fstream": "1.0.12",
```

Comment:
This was a transient dependency that we manually declared & updated due to an issue reported in Sourceclear - ideally I'd like to remove these and have webpack manage its own transient dependencies.

Reply:
Makes sense, resolved in 87d7ac1.  I manually confirmed it compiles, and visualizations still work.